packages/components/psammead-idt1-include/src/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import { Helmet } from 'react-helmet';
+
+const Idt1Include = ({ html, requireJsSrc }) => {
+  const [isRequireLoaded, setIsRequireLoaded] = useState(false);
+
+  const handleOnLoad = () => {
+    setIsRequireLoaded(true);
+  };
+
+  return (
+    <>
+      <Helmet>
+        <script async onLoad={handleOnLoad} src={requireJsSrc} />
+      </Helmet>
+      {isRequireLoaded && <div dangerouslySetInnerHTML={{ __html: html }} />}
+    </>
+  );
+};
+
+export default Idt1Include;
```

Comment:
Nearly identical to @simonsinclair 's implementation on the ticket:
- It waits for the requirejs script to be loaded asyncronously
- Then rerenders, to add the include payload; this running any inline scripts that utilise require.